packages/services/src/services/Locale/__tests__/Locale.test.js

```diff
@@ -12,6 +12,7 @@ jest.mock('@carbon/ibmdotcom-utilities', () => ({
   },
   geolocation: jest.fn(() => Promise.resolve('us')),
 }));
+jest.setTimeout(20000);
```

Comment:
What is this doing here?

Reply:
leaving it the default timeout of 5000 ms didn't give it enough the test enough time to wait for the instances of ```const lang = await LocaleAPI.getLang();``` to complete when there was no html lang attr and no DDO. it seemed to need an average of ~12 seconds.